src/frontends/tensorflow_common/src/op/case.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "input_model.hpp"
+#include "openvino/opsets/opset10.hpp"
```

Comment:
please avoid usage of opset header. Use operation header instead like `openvino/op/if.hpp`.